source/renderer/app/components/wallet/wallet-import/WalletSelectImportDialog.js

```diff
@@ -135,6 +135,25 @@ export default class WalletSelectImportDialog extends Component<Props> {
     intl: intlShape.isRequired,
   };
 
+  handleSelfRef = (ref: ?ElementRef<'div'>) => {
+    if (ref) {
+      this.applyDialogStyles();
+    }
+  };
+
+  applyDialogStyles = () => {
+    const dialogElement = window.document.querySelector('.ReactModal__Content');
+    const dialogOverlayElement = dialogElement.parentElement;
+    dialogOverlayElement.style.backgroundColor =
+      'var(--theme-wallet-import-background-color)';
+    dialogElement.style.backgroundColor = 'transparent';
+    dialogElement.style.border = 'none';
+    dialogElement.style.boxShadow = 'none';
+    dialogElement.style.height = '100%';
+    dialogElement.style.width = '100%';
+    dialogElement.style.maxWidth = 'initial';
+  };
+
```

Comment:
@DeeJayElly same comment as above - styles outside jsx and refs

Reply:
Well i tried to override it without this approach with JSX, but unfortunately i couldn't find a way to do it since this ReactModal is basically React Polymorph component.

Reply:
https://github.com/input-output-hk/daedalus/pull/2014#discussion_r423675987

Reply:
Implemented